lint/problem.go

```diff
@@ -106,25 +106,25 @@ func (p Problem) GetRuleURI() string {
 	return getRuleURL(string(p.RuleID), ruleURLMappings)
 }
 
-// position describes a one-based position in a source code file.
+// Position describes a one-based Position in a source code file.
 // They are one-indexed, as a human counts lines or columns.
-type position struct {
+type Position struct {
 	Line   int `json:"line_number" yaml:"line_number"`
 	Column int `json:"column_number" yaml:"column_number"`
 }
 
-// fileLocation describes a location in a source code file.
+// FileLocation describes a location in a source code file.
 //
 // Note: Positions are one-indexed, as a human counts lines or columns
 // in a file.
-type fileLocation struct {
-	Start position `json:"start_position" yaml:"start_position"`
-	End   position `json:"end_position" yaml:"end_position"`
+type FileLocation struct {
+	Start Position `json:"start_position" yaml:"start_position"`
+	End   Position `json:"end_position" yaml:"end_position"`
 }
 
-// fileLocationFromPBLocation returns a new fileLocation object based on a
+// FileLocationFromPBLocation returns a new fileLocation object based on a
 // protocol buffer SourceCodeInfo_Location
-func fileLocationFromPBLocation(l *dpb.SourceCodeInfo_Location) fileLocation {
+func FileLocationFromPBLocation(l *dpb.SourceCodeInfo_Location) FileLocation {
```

Comment:
I realize now that this is more or less directly reversing a deliberate decision in #195 to not have two types of "location." If that's still desirable then maybe the other path of fixing the math directly in the github formatter, or possibly a third option of trying to centralize the "formatter" concept somewhere they could share utilities like this